src/redux/actions/concepts/addBulkConcepts/index.js

```diff
@@ -14,7 +14,7 @@ import {
 } from '../../types';
 
 export const fetchBulkConcepts = (currentPage, source = 'CIEL') => async (dispatch) => {
-  const url = `orgs/${source}/sources/${source}/concepts/?limit=10&page=${currentPage}&&verbose=true`;
```

Comment:
Is there any reason we are removing the limit on this?

Reply:
The endpoint by default is paginated to return 10 results. Hence making `?limit=10` redundant

Reply:
Noted.